pkg/maps/vtep/logfields.go

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2018 Authors of Cilium
+
+package vtep
+
+import (
+	"github.com/cilium/cilium/pkg/logging"
+	"github.com/cilium/cilium/pkg/logging/logfields"
+)
+
+const (
+	fieldEndpoint = "endpoint"
+	fieldPrefix   = "prefix"
+	fieldMac      = "mac"
```

Comment:
This is good style if the fields are unique to this package, but for each of these fields there is already a canonical definition in `pkg/logging/logfields`. Could you use those instead?

Reply:
done